Source/Particles/Collision/BackgroundMCCCollision.H

```diff
@@ -0,0 +1,73 @@
+/* Copyright 2021 Roelof Groenewald
+ *
+ * This file is part of WarpX.
+ *
+ * License: ????
+ */
+#ifndef WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+#define WARPX_PARTICLES_COLLISION_BACKGROUNDMCCCOLLISION_H_
+
+#include "Particles/MultiParticleContainer.H"
+#include "CollisionBase.H"
+#include "MCCProcess.H"
+#include <AMReX_ParmParse.H>
```

Comment:
Please add also for the other AMReX classes and STL types/algorithms used in this file:
- ManagedVector
- Real
- string

I think ParamParse is actually not used here?

```suggestion

#include <AMReX_REAL.H>
#include <AMReX_GpuContainers.H>

#include <string>
```